packages/data-explorer-ui/src/providers/exploreState.tsx

```diff
@@ -203,6 +219,16 @@ export function ExploreStateProvider({
       exploreReducer(s, a, { config, entityConfig }),
     {
       categoryViews: [],
+      entityPageState: config.entities.reduce(
+        (acc, { list: { columns }, route }) => ({
+          ...acc,
+          [route]: {
```

Comment:
I wonder if we can get some reuse out of https://github.com/clevercanary/data-explorer/blob/775f22cc75643348a99f752de181f94a154cb640/packages/data-explorer-ui/src/components/Table/common/utils.ts#L247?

If we grab `defaultSort` from line `223` e.g.  update `223` with:

```(acc, { list: { columns, defaultSort }, route }) => ({```

and then update lines `225-228` with:

```
[route]: getInitialState(columns, defaultSort)
```

Let me know if this is suitable re-use?